src/services/PaymentService.ts

```diff
@@ -447,15 +478,31 @@ export class PaymentService {
   private async findIndividualActiveSubscription(customerId: CustomerId): Promise<Subscription> {
     const activeSubscriptions = await this.getActiveSubscriptions(customerId);
 
-    const individualActiveSubscription = activeSubscriptions.find(
-      (subscription) => subscription.items.data[0].price.metadata.is_teams !== '1',
-    );
+    const individualActiveSubscription = activeSubscriptions.find((subscription) => {
+      const isNotTeams = subscription.items.data[0].price.metadata.is_teams !== '1';
```

Comment:
The teams thing is no longer needed, you can remove this line (this was the old b2b solution)